esphome/components/ili9xxx/display.py

```diff
@@ -60,6 +66,17 @@ def _validate(config):
         raise cv.Invalid(
             "Providing color palette images requires palette mode to be 'IMAGE_ADAPTIVE'"
         )
+    if CORE.is_esp8266 and config.get(CONF_MODEL) not in [
```

Comment:
I think this should be opposite
```suggestion
    if CORE.is_esp8266 and config.get(CONF_MODEL) in [
```

Reply:
As talked about on #BETA on Discord.
It was half true. Only the old models from ili9341 driver sould in the list.